roles/openshift_etcd_client_certificates/tasks/main.yml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
Let's remove this.  Just change where ever is calling role: openshift_etcd_client_certificates to use include_role, if possible.

Reply:
I can't do it right now due to https://github.com/ansible/ansible/issues/21890. I would need to replace the list of roles [1] with a list of include_task. There is at least one role in the list the expose its role variable which are used further in other role. If I replace the `role` with `include_task`, the variables from the defaults and vars will not get exposed to the play and the play will fail.

[1] https://github.com/openshift/openshift-ansible/blob/master/playbooks/common/openshift-master/config.yml#L190-L201